### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by bnb** [slate/slate#79]
I was wondering if someone could tell me how to get Slate to work on Windows. Could anybody? I tried for about 45 minutes to get GNU `make` installed on my Windows 8 computer, but none of the methods I found worked. Is there another way to build it on windows?

**Comment by looterz** [comment]
Same issue here, couldn't get it setup properly.

**Comment by tj** [comment]
changed the "and Windows" to "and eventually Windows" since it should theoretically work fine, but I haven't tried

**Comment by bnb** [comment]
Just to let you know, since you haven't tried it, I tried the methods mentioned [here](https://github.com/rogerwang/node-webkit/wiki/How-to-run-apps) for All Platforms and Windows, and both resulted in the following error in a node-webkit browser window: 

```
TypeError: Property 'slate' of object [object Object] is not a function
    at eval (file:///C:/path/to/slate/slate-master/index.html:40:7)
```

Maybe if this is fixed it will work?

**Comment by looterz** [comment]
That's the same error I was getting bnb.

**Comment by yasoob** [comment]
@bnb I also got the same error today. Still trying to solve it. Did you manage to solve it?

**Comment by bnb** [comment]
@yasoob No, I didn't, unfortunately. I didn't really attempt to, though.

**Comment by yasoob** [comment]
Agh! I really like this app and really want to get my hands on it. Pity me :(

**Comment by tj** [comment]
To be fair you'd probably be disappointed right now, there's still quite a bit of work we need to make things fully functional

**Comment by bnb** [comment]
@visionmedia @yasoob Totally understand. Looking forward to when this is fixed.

**Comment by Acconut** [comment]
Today I tried to run slate on Windows.

The make targets don't work so I did the build task manually (after cloning slate and copying node-webkit):

```
npm install
node_modules\.bin\component.cmd install
node_modules\.bin\component.cmd build --standalone slate
zip -rq slate.nw *
```

Windows doesn't have a `zip` command by default as Unix has, so I used http://stahlworks.com/dev/?tool=zipunzip which supports the same arguments.

You will now have a slate.nw and nw.exe file in your folder.

To start the app you need a command line in administrator mode and run:

```
nw slate.nw
```

If you don't run it as admin you'll get following error in an alert:

> failed to load plugin "light": Error: EPERM: symlink
> D:\Programs\slate\plugins\light\icons\font\slate.eot

Now a window should open.
![image](https://cloud.githubusercontent.com/assets/1375043/2618135/12025308-bc1c-11e3-9662-324605081d73.png)

The controls on the upper right work, but nothing else does.
The console prints following error:

```
GET file:///D:/Programme/Slate/slate/light/icons/font/slate.svg?93118699  light/icons/font/slate.svg?93118699#slate:1
```

I hope this might help and we can run it on windows some time. :)
